### ISSUE TITLE
Issue with SELU

### ISSUE THREAD

**Issue by discordance** [onnx/keras-onnx#163]
With model using SELU as activation layers, the params Alpha and Gamma defaults are set to None, which trigger a ValueError

**Comment by jiafatom** [comment]
Can you share the model please?

**Comment by discordance** [comment]
I am using keras via tf.keras v1.14

`
#create model
model = keras.models.Sequential()

# l1
model.add(keras.layers.Conv2D(32, strides=(2,2), kernel_size=3, input_shape=(SIZE,SIZE,1)))
model.add(keras.layers.BatchNormalization())
model.add(keras.layers.Activation('selu'))
model.add(keras.layers.Conv2D(32, strides=(2,2), kernel_size=3, input_shape=(SIZE,SIZE,1)))
model.add(keras.layers.BatchNormalization())
model.add(keras.layers.Activation('selu'))
model.add(keras.layers.MaxPooling2D((2,2), padding='same'))
# model.add(keras.layers.Dropout(0.2))

model.add(keras.layers.Flatten())
model.add(keras.layers.Dense(32, activation='selu'))
# model.add(keras.layers.Dropout(0.5))
model.add(keras.layers.Dense(NB_CLASS, activation='softmax'))

model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])
`

**Comment by jiafatom** [comment]
There is a bug in onnxconverter-common and I checked in the [fix](https://github.com/microsoft/onnxconverter-common/pull/10). Before its next release, please pull onnxconverter-common master or hotfix on your local branch.

**Comment by jiafatom** [comment]
onnxconverter-common 1.5.1 is published in pypi. Please upgrade the package.
